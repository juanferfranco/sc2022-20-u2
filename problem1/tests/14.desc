Crea un evento con un nombre de mÃ¡s de 15 caracteres