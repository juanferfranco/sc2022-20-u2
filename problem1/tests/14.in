add
ev1234567890qwertyuiop
ask
exit
