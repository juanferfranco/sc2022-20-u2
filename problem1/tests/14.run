./main < tests/14.in
