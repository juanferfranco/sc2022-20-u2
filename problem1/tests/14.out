ev1234567890qwe
